packages/chrome-extension/src/internal/utils/__tests__/manifest.test.ts

```diff
@@ -1,13 +1,7 @@
 import type { Manifest } from 'webextension-polyfill';
 
-import { missingManifestKeyError, missingValidManifestHostPermission } from '../errors';
-import type { ValidatedManifest } from '../manifest';
-import { getValidPossibleManifestHosts, validateHostPermissionExistence, validateManifest } from '../manifest';
-
-const validClerkManifest = {
-  permissions: ['cookies', 'storage'],
-  host_permissions: ['http://localhost:3000'],
-} as ValidatedManifest;
+import { missingManifestKeyError } from '../errors';
+import { validateManifest } from '../manifest';
 
 describe('Manifest', () => {
   describe('validateManifest(manifest)', () => {
```

Comment:
Reduced complexity of the `validateManifest` thus these tests are no longer necessary.